modules/account-lib/src/coin/algo/assetTransferBuilder.ts

```diff
@@ -69,6 +71,12 @@ export class AssetTransferBuilder extends TransferBuilder {
       this.suggestedParams,
       this._reKeyTo,
     );
+    if (!_.isUndefined(this._lease)) {
```

Comment:
Move this to a parent class and refactor. This function must be on txBuilder.

Reply:
Done!